### PR TITLE
Hotfix/112 add food checkbox on the shopping sheet

### PR DESCRIPTION
**Pull request by lmeng25**
Added food checkbox on the shopping sheet
image

Added food label on visit form, and visit details
image
image

#112

### CODE REVIEW

File: models/index.d.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ export declare type Visit = {
     readonly clothingBoy?: boolean | null;
     readonly clothingMen?: boolean | null;
     readonly sleepingBag?: boolean | null;
+    readonly food?: boolean | null;
```
Comment (ducmvo) [inline models/index.d.ts:48]: I'm wondering do we need a food items quantity?

Comment (lmeng25) [reply models/index.d.ts:48]: That's a good question. Shall we ask the client to clarify before pushing the changes?

Comment (lmeng25) [reply models/index.d.ts:48]: The client confirmed that food quantity is unnecessary on the 11/1 meeting.

**Review by ducmvo — APPROVED** [review]

**Comment by lmeng25** [comment]
Close and merge to /dev instead.

**Pull request closed without merging**